README.md

```diff
@@ -135,6 +135,31 @@ This will list all the alerts belonging to that incident.
                        (default "incident.id,alert,cluster.name,cluster.id,status,severity")
 ```
 
+## Oncall
+
+To view the current oncalls as per PagerDuty, use the command:
+
+```
+pdcli oncall
+```
+### Oncall View Navigation
+
+By default, all the escalations and Oncalls are displayed for team **Platform-SRE** in the main view.
+
+You can choose to view the Oncalls for each Escalation for all teams on PagerDuty using **All Teams Oncall**.
+
+To view your Oncall schedule, use **Your Next Oncall Schedule**.
```

Comment:
I feel this can be omitted because you're explaining the same in the nav table.

Reply:
omitted